src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Card, CardContent, CardHeader, Grid, Typography
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  bold: {
+    fontWeight: 'bold'
+  }
+});
+
+const HealthCardDisplay = ({ patientData }) => {
+  const classes = useStyles();
+
+  const immunizationCode = (codings) => {
+    if (codings.length === 0) return '';
+
+    const coding = codings[0];
+    return coding.system ? `${coding.system}#${coding.code}` : coding.code;
+  };
+
+  return (
+    <div>
+      <Typography gutterBottom>
+        <span className={classes.bold}>Name: </span>
+        <span>{patientData.name}</span>
+      </Typography>
+      <Typography gutterBottom>
+        <span className={classes.bold}>Date of Birth: </span>
+        <span>{patientData.dateOfBirth}</span>
+      </Typography>
+      <Typography variant="h4"> Immunizations </Typography>
+      <Grid
+        container
+        spacing={2}
+        direction="row"
+        justifyContent="flex-start"
+        alignItems="flex-start"
+      >
+        {patientData.immunizations.map((immunization) => (
+          <Grid item key={`${immunization.lotNumber}`}>
```

Comment:
`lotNumber` is not required and also not unique, so it is causing duplicate key warnings in the console. Could we change this key to something more unique? Maybe the each immunization's `fullUrl` element could also be returned from `extractImmunizations()`. Or you could probably just do something simpler with indices or random values.

Reply:
Lint complains about using index for the key and random values are not good since React expects stable keys. I changed it to use the `fullUrl`.

Reply:
Now an error is being thrown for indexing a performer that doesn't exist:
![Screen Shot 2021-08-23 at 12 26 47 PM](https://user-images.githubusercontent.com/51176978/130483472-aa112f1e-c3ba-4bf3-8c7f-c9424b0a1028.png)

Reply:
Should be fixed.